src/main/java/uk/nhs/tis/trainee/actions/service/ActionService.java

```diff
@@ -104,6 +125,7 @@ public List<ActionDto> updateActions(Operation operation, PlacementDto dto) {
     }
 
     if (deleteAction) {
+      Action action = mapper.toAction(dto, REVIEW_DATA);
```

Comment:
since `deleteIncompleteActions` uses the `action` as a template, it doesn't matter what specific `actionType` is assigned to it. Its not intuitive though.